src/ansible_compat/runtime.py

```diff
@@ -238,7 +238,14 @@ def warning(
     def _add_sys_path_to_collection_paths(self) -> None:
         """Add the sys.path to the collection paths."""
         if not self.isolated and self.config.collections_scan_sys_path:
-            self.config.collections_paths.extend(sys.path)  # pylint: disable=E1101
+            for path in sys.path:
+                if (
+                    path not in self.config.collections_paths
+                    and Path(Path(path) / "ansible_collections").is_dir()
```

Comment:
I think you may have an extra "Path" here :)

Reply:
updated